esp/esdllib/esdl_def.cpp

```diff
@@ -1461,6 +1462,8 @@ IEsdlDefObjectIterator* EsdlDefinition::getDependencies( const char* service, St
         methodIter.setown(serviceDef->getMethods());
         for( methodIter->first(); methodIter->isValid(); methodIter->next() )
         {
+            if ((flags & DEPFLAG_ECL_ONLY) && methodIter->query().getPropInt("ecl_hide"))
```

Comment:
That would be fine, but should be a new issue.  Here I just want to be consistent with existing code.